test/system/src/test/java/io/pravega/test/system/ReadWriteAndAutoScaleWithFailoverTest.java

```diff
@@ -100,6 +100,8 @@ public void setup() {
 
     @After
     public void tearDown() {
+        testState.stopReadFlag.set(true);
```

Comment:
In the test run that was the impetus for this PR, there was a lot of log noise at the end because this was not done.